docs/getting-started/cli.md

```diff
@@ -0,0 +1,198 @@
+---
+slug: cli
+title: CLI-only workflow (Pro)
+hide_title: true
+description: Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+tags:
+  - quickstart
+  - cli
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+import Install from "/src/components/procedure/_install-cli.mdx";
+import Login from "/src/components/procedure/_login-activate.mdx";
+
+# CLI-only workflow with Semgrep's Pro Engine
```

Comment:
```suggestion
# Local scans with Semgrep Pro Engine
```
I appreciate the callout "with Semgrep Pro Engine" and happy we provide them with the OSS flag.